LAKMobile/src/components/ConfirmationBox.tsx

```diff
@@ -0,0 +1,141 @@
+/**
+ * Apply to job?
+ * 
+ * Be sure to contact <Client> at <phone number>
+ * 
+ * cancel, apply
+ */
```

Comment:
I don't think we need this comment here anymore, so can we remove it and add something that's more descriptive of what the confirmation box does?